adoc/chapters/opencl_backend.adoc

```diff
@@ -1055,9 +1055,10 @@ C libraries during offline compilation or during online compilation by the
 
 Linking with OpenCL C kernel functions offline is an optional feature and is
 unspecified.
-Linking with OpenCL C kernel functions online is performed by using the SYCL
-[code]#kernel_bundle# class to compile and link an OpenCL C source; using the
-[code]#compile_with_source# or [code]#build_with_source# member functions.
+Linking with OpenCL C kernel functions online is performed via the
+[code]#make_kernel_bundle# or [code]#create_kernel_bundle# functions, which both
```

Comment:
```suggestion
[code]#sycl::make_kernel_bundle# or [code]#sycl::opencl::create_bundle# functions, which both
```

Note that the two functions are in different namespaces, so it's probably good to show the fully qualified names here.